src/boot/efi/devicetree.h

```diff
@@ -9,6 +9,28 @@ struct devicetree_state {
         void *orig;
 };
 
+enum {
+        FDT_BEGIN_NODE = 1,
+        FDT_END_NODE   = 2,
+        FDT_PROP       = 3,
+        FDT_NOP        = 4,
+        FDT_END        = 9,
+};
+
+struct fdt_header {
+        uint32_t Magic;
```

Comment:
struct names should be in `PascalCase` and fields should be in `snake_case`. Also, you should typedef the struct definitions to avoid writing `struct` everywhere. So in this case:

```c
typedef struct FdtHeader {
        uint32_t magic;
        uint32_t total_size;
        uint32_t off_dt_struct;
        uint32_t off_dt_strings;
        uint32_t off_mem_rsv_map;
        uint32_t version;
        uint32_t last_comp_version;
        uint32_t boot_cpuid_phys;
        uint32_t size_dt_strings;
        uint32_t size_dt_struct;
} FdtHeader;
```

Please adjust the rest of the patchset, and refer to `docs/CODING_STYLE.md` for code style requirements of systemd

Reply:
Hopefully it's fine now. Maybe there should be a CI pipeline for checking the code style and/or proper .clang-format which can be used for autoformatting?